### PR TITLE
FLEX-593 Add node 12

### PR DESCRIPTION
**Pull request by galexandrov**

### CODE REVIEW

**Comment by galexandrov** [comment]
All tests are passing, but the security check is failing and can't be fixed
```
npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Regular Expression Denial of Service                         │
├──────────���────┼──────────────────────────────────────────────────────────────┤
│ Package       │ string                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ suppose [dev]                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ suppose > string                                             │
├───────────────┼────────────────────────────────���─────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/536                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 high severity vulnerability in 1470 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

File: package.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "eslint-plugin-import": "2.13.0",
     "express": "4.16.2",
     "mocha": "5.0.4",
-    "nyc": "13.1.0",
+    "nyc": "^14.1.1",
```
Comment (DennyGD) [inline package.json]: Is there a specific reason to update? Not complaining, just asking 😄 
Perhaps, choose a specific version (without the `^`)?

Comment (galexandrov) [reply package.json]: This change was caused by ``npm audit fix``, I will revert it

Comment (galexandrov) [reply package.json]: I will revert the changes for ``package.lock`` too

**Pull request merged**